testnet/stacks-node/src/event_dispatcher.rs

```diff
@@ -46,7 +46,7 @@ impl EventObserver {
         };
 
         let url = {
-            let url = format!("{}{}", &self.endpoint, path);
+            let url = format!("http://{}/{}", &self.endpoint, path);
```

Comment:
That would only be the case if users are proving `listener:3700/` in the config, instead of `listener:3700`, right?

Reply:
`path` could start with `/`.

Reply:
yeah, I was assuming that we were using these constants for `path`:
```
pub const PATH_MEMPOOL_TX_SUBMIT: &str = "new_mempool_tx";
pub const PATH_BLOCK_PROCESSED: &str = "new_block";
```
Will address.
```

Reply:
No -- I think the config will add it always:

https://github.com/blockstack/stacks-blockchain/blob/master/testnet/stacks-node/src/config.rs#L338

This was so that people could configure it with or without that trailing slash.

This change _also_ would prevent someone from using https if they were running their endpoint observer somewhere other than the same network, but if the node doesn't work when the config specifies the HTTP scheme, then it's fine for it to always be HTTP.

Reply:
Ha, ok, missed the "/" tweak, thank you for the pointer.
I thought about the HTTP/HTTPS limitation, but I don't think that we've planned supporting it.

Reply:
Addressed with https://github.com/blockstack/stacks-blockchain/pull/1718/commits/fef763a63e1504035bf85387ba8dbb62a245ddfb